dali/kernels/signal/decibel/decibel_calculator.h

```diff
@@ -66,6 +73,30 @@ struct DecibelCalculator {
 };
 
 
+template <typename T>
+struct ToDecibelCalculator : public DecibelCalculator<T> {
```

Comment:
actually, the inheritance is reverse of what it should be.

Reply:
fixed